tests/components/wled/test_light.py

```diff
@@ -162,10 +162,13 @@ async def test_light_connection_error(
     hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
 ) -> None:
     """Test error handling of the WLED switches."""
-    aioclient_mock.post("http://192.168.1.123:80/json/state", exc=aiohttp.ClientError)
     await init_integration(hass, aioclient_mock)
 
-    with patch("homeassistant.components.wled.WLEDDataUpdateCoordinator.async_refresh"):
+    with patch(
+        "homeassistant.components.wled.WLEDDataUpdateCoordinator.async_refresh"
```

Comment:
Can we patch `WLED.update` instead and reach the same goal?

Reply:
Well, no. As the service call hits the `light` method, which should throw a WLEDConnectionError.

The `async_refresh` should be a NOOP. If that would throw the same, it would be hard to determine if the tests, in this case, succeeded because of the refresh happening or the service call.

Reply:
Done